build.gradle.kts

```diff
@@ -337,6 +337,14 @@ project(":plugin") {
                 into("${pluginName.get()}/prettyPrinters")
                 include("*.py")
             }
+            from("$rootDir/prettyPrinters/gdb_formatters") {
+                into("${pluginName.get()}/prettyPrinters/gdb_formatters")
+                include("*.py")
+            }
+            from("$rootDir/prettyPrinters/lldb_formatters") {
+                into("${pluginName.get()}/prettyPrinters/lldb_formatters")
+                include("*.py")
+            }
```

Comment:
It seems you can just replace initial `include("*.py")` with `include("**/*.py")` and avoid these new `from` blocks